initial_tests.ipynb

```diff
@@ -182,7 +183,9 @@
     "                continue # Dont over add\n",
     "\n",
     "            count_per_class[label] += 1\n",
-    "\n",
+    "            if count_per_class[label] == min_necessary_classes:\n",
+    "                enough_samples_per_class[label] = True\n",
+    "                \n",
```

Comment:
Is the behavior identical?

Reply:
Oh, yes, you are right! Don't know why I got confused there

Reply:
No worries, just wondering if there was a bug invalidating my results. Ill PR. Thanks Oleg.